Client/package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.6.3",
     "axios": "^0.21.1",
+    "express": "^4.17.1",
```

Comment:
Express should not exist in client directory

Reply:
removed